tools/rdf_generator/setup.py

```diff
@@ -25,6 +25,6 @@
     author_email='charbelk@google.com',
     description='',
     packages=find_packages(),
-    install_requires=['absl-py', 'pyglib', 'ruamel.yaml', 'rdflib==5.0.0'],
+    install_requires=['absl-py', 'pyglib', 'ruamel.yaml==0.17.4', 'rdflib==5.0.0'],
```

Comment:
since you touched this file, it seems the linter is complaining about a line too long, can you please check.